UnityProject/Assets/Scripts/Items/ThrowableBreakable.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AddressableReferences;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace Items
+{
+	public class ThrowableBreakable : MonoBehaviour
+	{
+		[SerializeField]
+		private GameObject brokenItem = default;
+
+		[SerializeField, Range(0, 100)]
+		private int chanceToBreak = 100;
+
+		[SerializeField]
+		private AddressableAudioSource soundToPlay = SingletonSOSounds.Instance.GlassBreak01;
+
+		private CustomNetTransform customNetTransform;
+
+		private void Start()
+		{
+			customNetTransform = GetComponent<CustomNetTransform>();
+			customNetTransform.OnThrowEnd.AddListener(MyListener);
+		}
+
+		private void OnDisable()
+		{
+			customNetTransform.OnThrowEnd.RemoveListener(MyListener);
+		}
+
+		private void OnThrown(ThrowInfo info)
```

Comment:
Method was renamed to `OnThrown` but lines 25, 30 weren't updated to reflect the changes.